attribute.go

```diff
@@ -24,6 +25,29 @@ type Attribute struct {
 	Data []byte
 }
 
+// IsNested interprets the Type field of the Attribute to determine whether it's
+// a nested attribute or not. This facilitates recursive parsing of the attribute.
+// This is the leftmost bit in Type. Mutually exclusive with IsNetByteOrder().
+func (a Attribute) IsNested() bool {
+	return (a.Type & unix.NLA_F_NESTED) > 0
+}
+
+// IsNetByteOrder interprets the Type field of the Attribute to determine whether the attribute
```

Comment:
Out of curiosity, what part of the attribute would be big endian? Is it just the data portion?